evergreen.yml

```diff
@@ -654,36 +786,69 @@ tasks:
     commands:
       - func: "check rustfmt"
 
-  - name: compile
+  - name: windows-compile
     commands:
       - func: "set packages version"
       - func: "compile release"
       - func: "mciuploads release build and symbols"
 
-  - name: compile-debug
+  - name: windows-compile-debug
     commands:
       - func: "set packages version"
       - func: "compile debug"
       - func: "mciuploads debug build and symbols"
 
-  - name: unit-test
+  - name: windows-unit-test
     commands:
       - func: "install rust toolchain"
-      - func: "run unit tests"
+      - func: "run windows unit tests"
 
-  - name: integration-test
+  - name: windows-integration-test
     depends_on:
-      - name: compile
+      - name: windows-compile
     commands:
       - func: "install rust toolchain"
-      - func: "run integration tests"
+      - func: "run windows integration tests"
 
-  - name: result-set-test
+  - name: windows-result-set-test
     depends_on:
-      - name: compile
+      - name: windows-compile
     commands:
       - func: "install rust toolchain"
-      - func: "run result set test"
+      - func: "run windows result set test"
+
+  - name: ubuntu-compile
+    commands:
+      - func: "install unix odbc"
+      - func: "set packages version"
+      - func: "compile release"
+      - func: "mciuploads release build and symbols"
+
+  - name: ubuntu-compile-debug
+    commands:
+      - func: "install unix odbc"
+      - func: "set packages version"
+      - func: "compile debug"
+      - func: "mciuploads debug build and symbols"
+
+  - name: ubuntu-unit-test
+    commands:
+      - func: "install unix odbc"
+      - func: "run ubuntu unit tests"
+
+  - name: ubuntu-integration-test
+    commands:
+      - func: "install unix odbc"
+      - func: "compile release with debug info"
```

Comment:
The other versions without debug info are wrong, and I was too lazy to fix them.

Reply:
I'm only using the debug info ones in tests. I think it's fine.

Reply:
Functionally yes. For integration tests, I'd use the release version to try to avoid having surprise segfault when users are using the release version vs the release with debug because of the small difference.
Not a big deal, just a little extra safety, that's why I mentioned it.

Reply:
Functionally yes. For integration tests, I'd use the release version to try to avoid having surprise segfault when users are using the release version vs the release with debug because of the small difference.
Not a big deal, just a little extra safety, that's why I mentioned it.